flight/UAVObjects/uavobjecttemplate.c

```diff
@@ -74,28 +98,10 @@ int32_t $(NAME)Initialize(void)
  */
 void $(NAME)SetDefaults(UAVObjHandle obj, uint16_t instId)
 {
-	$(NAME)Data data;
+	$(NAME)SetDefaultsImpl(obj, instId);
 
-	// Initialize object fields to their default values
-	UAVObjGetInstanceData(obj, instId, &data);
```

Comment:
I'm confused by this function. It seems like it sets the UAVO data to all-zero; where are the actual defaults set?